### PR TITLE
Fixed compatibility with doctrine/dbal 3

### LINKED ISSUE #402: Incompatibility with doctrine/dbal
Hello,

In `Lexik\Bundle\TranslationBundle\Storage\DoctrineORMStorage` there is a method `getDriver()->getName()`. `getName()` method is deprecated in `doctrine/dbal` package and is not compatible with `3.0` version of `doctrine/dbal`. BC break described here:
https://github.com/doctrine/dbal/blob/3.1.x/UPGRADE.md

### PR DESCRIPTION
**Pull request by Knallcharge**
fixes #402

### CODE REVIEW

File: Storage/DoctrineORMStorage.php [hunk 1]
```diff
@@ -25,7 +26,7 @@ public function translationsTablesExist()
         $connection = $em->getConnection();
 
         // listDatabases() is not available for SQLite
-        if ('pdo_sqlite' !== $connection->getDriver()->getName()) {
+        if (get_class($connection->getDriver()) !== SQLiteDriver::class) {
```
Comment (bartmcleod) [inline Storage/DoctrineORMStorage.php:29]: I would use `instanceof here`. Is that not possible in this case?

Comment (Knallcharge) [reply Storage/DoctrineORMStorage.php:29]: Yes, works the same.

Comment (Seb33300) [reply Storage/DoctrineORMStorage.php:29]: Is this a BC break?

File: Storage/DoctrineORMStorage.php [hunk 2]
```diff
@@ -25,7 +26,7 @@ public function translationsTablesExist()
         $connection = $em->getConnection();
 
         // listDatabases() is not available for SQLite
-        if ('pdo_sqlite' !== $connection->getDriver()->getName()) {
+        if (!$connection->getDriver() instanceof SQLiteDriver::class) {
```
Comment (Seb33300) [inline Storage/DoctrineORMStorage.php:29]: `::class` is not required for `instanceof`

Comment (Knallcharge) [reply Storage/DoctrineORMStorage.php:29]: sorry, was too quick

File: Storage/DoctrineORMStorage.php (hunk 1 above)
Comment (andriusvo) [reply Storage/DoctrineORMStorage.php]: I think no?

**Review by andriusvo — APPROVED** [review]

**Pull request merged**